### ISSUE TITLE
Block meta bugs instead of depending on them

### ISSUE THREAD

**Issue by mb** [jgraham/site-report-triage#5]
Hi, I've noticed that new bugs in `Web Compatibility :: Privacy: Site Reports` depend on the meta [tp-breakage](https://bugzilla.mozilla.org/show_bug.cgi?id=1101005) or [etp-breakage](https://bugzilla.mozilla.org/show_bug.cgi?id=1480137). However, the convention is to block meta bugs. I've had trouble to set bugs as blocking sub-categories (e.g. [tp-breakage-cmp](bugzilla.mozilla.org/show_bug.cgi?id=1899334) due to circular dependency in bugzilla.

Description in [the wiki](https://wiki.mozilla.org/BMO/UserGuide/BugFields#dependson)

> Depends on: The bugs listed here must be resolved before this bug can be resolved.
> Blocks: This bug must be resolved before the bugs listed in this field can be resolved.

Relevant file: https://github.com/jgraham/site-report-triage/blob/7c1e580d35dd6e9b7cbd5c2d924151f5d39daee3/move-to-bugzilla.js#L206

Thanks!

**Comment by jgraham** [comment]
What we'd ideally like here is to move away from using meta bugs to indicate classes of issues. In particular the workflow that we're using for other site breakage bugs is:
* Keywords are used to label types of issues.
* Site report bugs depend-on the root cause core bugs that need to be resolved before the site is fixed.

We could of course special case the ETP bugs because they're not really a root cause like other core bugs. But I'd also say that having thousands of dependencies for a single bug doesn't work very well from the point of view of bugzilla performance or UI, so I'd encourage you to instead have keywords like, say, `privacy:etp-standard` and `privacy:etp-strict`.

**Comment by mb** [comment]
I agree that especially the [tp-breakage](https://bugzilla.mozilla.org/show_bug.cgi?id=1101005)-bug isn't that useful anymore. It is now should be equal to the `Web Compatibility :: Privacy: Site Reports` components. The sub meta bugs such as [tp-breakage-cmp](https://bugzilla.mozilla.org/show_bug.cgi?id=1899334) do make sense to me though. It makes it easier to modify all through the bugzilla interface. And write queries for them.
